src/main/scala/com/fulcrumgenomics/bam/Bams.scala

```diff
@@ -246,4 +248,34 @@ object Bams extends LazyLogging {
       }
     }
   }
+
+  /** Returns true if the read is mapped in an FR pair, false otherwise. */
+  def isFrPair(rec: SAMRecord): Boolean =
+    rec.getReadPairedFlag &&
+      !rec.getReadUnmappedFlag &&
+      !rec.getMateUnmappedFlag &&
+      rec.getReferenceIndex == rec.getMateReferenceIndex &&
+      SamPairUtil.getPairOrientation(rec) == PairOrientation.FR
```

Comment:
I am of the opinion of the statement isn't a one-liner there should be brackets.